### PR TITLE
Fresh start

### PR DESCRIPTION
**Pull request by keitwb**
Instead of replacing DDTracer with Jaeger's Java tracer, just change the DDTracer's output to be Zipkin V2 JSON to greatly minimize the diff from upstream.

This includes most of the customizations to the instrumentation and deployment we have made.

### CODE REVIEW

**Comment by tylerbenson** [comment]
Feel free to submit PRs for instrumentation improvements/bug fixes upstream too... 😉

**Comment by keitwb** [comment]
@tylerbenson We will!

**Comment by tylerbenson** [comment]
😇 We're also hiring if you'd rather just come *work* upstream instead...  You know how to find me. :trollface:

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 version: 2
 
 defaults: &defaults
-  working_directory: ~/dd-trace-java
+  working_directory: ~/signalfx-tracing-java
```
Comment (mpetazzoni) [inline .circleci/config.yml]: Shouldn't this be `signalfx-java-tracing` to match the repository name (everywhere)?

File: LICENSE [hunk 2]
```diff
@@ -187,6 +187,7 @@
       identification within third-party archives.
 
    Copyright 2017 Datadog, Inc.
+   Copyright 2018 SignalFx, Inc.
```
Comment (mpetazzoni) [inline LICENSE]: 2018-2019

**Pull request merged**